sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -293,20 +309,28 @@ export class EventHubConsumerClient {
         this._checkpointStore = checkpointStoreOrEventHubNameOrOptions3;
         this._userChoseCheckpointStore = true;
         this._clientOptions =
-          (checkpointStoreOrCredentialOrOptions4 as EventHubClientOptions) || {};
+          (checkpointStoreOrCredentialOrOptions4 as EventHubConsumerClientOptions) || {};
       } else {
         // 1
         this._checkpointStore = new InMemoryCheckpointStore();
         this._userChoseCheckpointStore = false;
         this._clientOptions =
-          (checkpointStoreOrEventHubNameOrOptions3 as EventHubClientOptions) || {};
+          (checkpointStoreOrEventHubNameOrOptions3 as EventHubConsumerClientOptions) || {};
       }
 
       this._context = createConnectionContext(
         connectionStringOrFullyQualifiedNamespace2,
         this._clientOptions
       );
     }
+    this._loadBalancingOptions = {
+      // default options
+      strategy: "balanced",
+      updateIntervalInMs: 10000,
+      partitionOwnershipExpirationIntervalInMs: 60000,
```

Comment:
Is 6 * updateIntervalInMs better than 60000 if a user passes in only updateIntervalInMs?

Reply:
I don't think so. In the API review, we specifically did not want to default the `partitionOwnershipExpirationIntervalIsMs` by some multiple of the `updateIntervalInMs`